web/src/components/FailureSwitcher.tsx

```diff
@@ -13,6 +22,12 @@ type PropsType = {
 const FailureSwitcher = ({ error }: PropsType): ReactElement => {
   const { t } = useTranslation('error')
 
+  useEffect(() => {
+    if (!(error instanceof NotFoundError) && !(error instanceof FetchError)) {
```

Comment:
why we can not use your created custom hook? useReportsError

Reply:
```suggestion
   useReportError(error)
```

Reply:
I only added that hook for native as it is only used once in web.